b2sdk/account_info/abstract.py

```diff
@@ -132,8 +132,9 @@ def is_master_key(self) -> bool:
         new_style_master_key_suffix = '0000000000'
         if account_id == application_key_id:
             return True  # old style
-        if len(application_key_id
-              ) == (3 + len(account_id) + len(new_style_master_key_suffix)):  # 3 for cluster id
+        if len(application_key_id) == (
+            3 + len(account_id) + len(new_style_master_key_suffix)
+        ):  # 3 for cluster id
```

Comment:
`yapf` vs `black` battle :) I prefer one line though

Reply:
me too, but if I had to choose, black is better